src/main/scala/scala/Xgettext.scala

```diff
@@ -125,21 +131,26 @@ msgstr ""
               val line       = (relPath(pos.source.path), pos.line)
 
               if (i18n_t.contains(methodName)) {
-                val msgid = fixBackslashSingleQuote(stringConstant(list.head, pos))
-                msgToLines.addBinding((None, msgid, None), line)
+                for (msgid <- stringConstant(list.head, pos)){
+                  msgToLines.addBinding((None, fixBackslashSingleQuote(msgid), None), line)
+                }
               } else if (i18n_tn.contains(methodName)) {
-                val msgid       = fixBackslashSingleQuote(stringConstant(list.head, pos))
-                val msgidPlural = fixBackslashSingleQuote(stringConstant(list(1), pos))
-                msgToLines.addBinding((None, msgid, Some(msgidPlural)), line)
+                for (msgid <- stringConstant(list.head, pos);
+                     msgidPlural <- stringConstant(list(1), pos)) {
+                  msgToLines.addBinding((None, fixBackslashSingleQuote(msgid), Some(fixBackslashSingleQuote(msgidPlural))), line)
+                }
               } else if (i18n_tc.contains(methodName)) {
-                val msgctxt = fixBackslashSingleQuote(stringConstant(list.head, pos))
-                val msgid   = fixBackslashSingleQuote(stringConstant(list(1), pos))
-                msgToLines.addBinding((Some(msgctxt), msgid, None), line)
+                for (msgctxt <- stringConstant(list.head, pos);
+                     msgid <- stringConstant(list(1), pos)){
```

Comment:
For style consistency, there should be a space between `)` and `{`. But that's OK.